### DISCUSSION TITLE
How to use dictionary in python code for a beginner, help wanted.

### DISCUSSION THREAD

**Discussion by Rolingmaniac** [community/community#128246] [Programming Help]
### Body

I am completely new to this and would appreciate any help with my problem, I would like to use a dictionary to try to decrypt a vigenere cipher, AI has written the code so I have no idea wha I’m doing, I have a dictionary.txt file I would like to use, but I can’t figure out how to link it to the code. Se pictures of how it looks now, what is the next step to link the two together?
![IMG_2571](https://github.com/community/community/assets/172282333/f3a91ffb-ed99-49a4-b838-3f73cb9383ec)
![IMG_2572](https://github.com/community/community/assets/172282333/35023eda-45a9-46d6-b00f-3d2b27babf3e)

### Guidelines

- [X] I have read and understood this category's [guidelines](https://github.com/orgs/community/discussions/51384) before making this post.

**Comment by DaviZCodes** [comment]
To read files in python, you need to open() them and then read() them. Basically, opening a file allows you to access it and reading just reads the contents of the file. 

In your code, the words variable are the words from the other file "dictionary.txt". They are essentially now linked together. 

You can learn more here: https://www.w3schools.com/python/ref_file_read.asp

**Comment by itsmichaelhagen** [comment]
Capitalize the "D" in the dictionary in your code to ensure it is correctly mapping to the txt file.
